pkg/webhook/policy.go

```diff
@@ -191,7 +210,18 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	}
 
 	res := resp.Results()
-	msgs := h.getDenyMessages(res, req)
+
+	gkDeploy := &appsv1.Deployment{}
+	deploy := types.NamespacedName{Namespace: util.GetNamespace(), Name: "gatekeeper-controller-manager"}
+	err = h.client.Get(ctx, deploy, gkDeploy)
+	if err != nil {
+		log.Error(err, "error getting gatekeeper deployment object")
+	}
+	ref, err := reference.GetReference(scheme.Scheme, gkDeploy)
```

Comment:
Is the reference expected to be stable? If so, we can cache it and move it out of the webhook to reduce latency and potential for failure.